frontend/app-development/features/processEditor/handlers/OnProcessTaskRemoveHandler.test.ts

```diff
@@ -1,13 +1,72 @@
 import type { Policy } from 'app-shared/types/Policy';
 import type { OnProcessTaskEvent } from '@altinn/process-editor/types/OnProcessTask';
-import type { TaskEvent } from '@altinn/process-editor/utils/ProcessTaskManager';
 import { OnProcessTaskRemoveHandler } from './OnProcessTaskRemoveHandler';
+import type { LayoutSets } from 'app-shared/types/api/LayoutSetsResponse';
+import { BpmnTypeEnum } from '@altinn/process-editor/enum/BpmnTypeEnum';
+import type { TaskEvent } from '@altinn/process-editor/types/TaskEvent';
+
+const orgMock = 'testOrg';
+const appMock = 'testApp';
```

Comment:
Maybe we can use the `org` and `app` from the studio/testing library? Imported like this: 
`import { app, org } from '@studio/testing/testids;`

Reply:
We could, but I find the test more readable when the value is not "org" and "app". 

```
 "urn:altinn:org:testOrg", //<-- quite easy to see that the testOrg is an variable 
 "urn:altinn:app:testApp" //<-- Same reason here
```

```
 "urn:altinn:org:org", //<-- otherwise it would be org:org
 "urn:altinn:app:app" //<-- and app:app here.
```

What do you think?

Reply:
I agree with you, but I think we should try to do it the same way through our entire repo. So, then I think it is better to change this: `import { app, org } from '@studio/testing/testids;` to this: `import { mockApp, mockOrg } from '@studio/testing/testids;`
In this way, we use the test variables the same way through all of our repo 😄 
What do you think?